src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -97,27 +97,27 @@ public static class TreeExtensions
         /// Ranks a series of inputs based on their relevance, training a decision tree ranking model through the <see cref="FastTreeRankingTrainer"/>.
         /// </summary>
         /// <param name="catalog">The <see cref="RankingCatalog"/>.</param>
-        /// <param name="labelColumn">The labelColumn column.</param>
-        /// <param name="featureColumn">The featureColumn column.</param>
-        /// <param name="groupId">The groupId column.</param>
-        /// <param name="weights">The optional weights column.</param>
+        /// <param name="labelColumnName">The name of the label column.</param>
+        /// <param name="featureColumnName">The name of the feature column.</param>
+        /// <param name="rowGroupColumnName">The name of the group column.</param>
```

Comment:
This isn't necessary for this PR, but I wonder if we need some documentation around the `row group` column. I'm obviously not an expert, but I have no idea what this is.

Reply:
fyi...#2536 gives some context on this.  we can add more documentation around this in  follow up documentation focused  PRs.